enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,39 @@
 const enhancer = require('./enhancer.js');
 // test away!
+
+describe('repair()', () => {
+    it('should restore durability to 100', () => {
+        const item = { durability: 55 };
+        const repair = enhancer.repair(item);
+
+        expect(repair.durability).toBe(100);
+    });
+});
+
+describe('succeed()', () => {
+    it('should enhance the item by 1', () => {
```

Comment:
change description to reflect the test, it `should not enhance the item by 1 when enhancement is 20`